Content.Client/GameObjects/Components/Kitchen/MicrowaveVisualizer.cs

```diff
@@ -38,8 +37,9 @@ public override void OnChangeData(AppearanceComponent component)
                     var audioParams = AudioParams.Default;
                     audioParams.Loop = true;
                     var schedSound = new ScheduledSound();
-                    schedSound.Filename = MicrowaveSoundLoop;
+                    schedSound.Filename = "/Audio/machines/microwave_loop.ogg";
                     schedSound.AudioParams = audioParams;
+                    _soundComponent.StopAllSounds();
                     _soundComponent.AddScheduledSound(schedSound);
```

Comment:
Finally, FINALLY someone using SoundComponent as intended! I'm proud of you, Memory. (This isn't really a review, so go ahead and resolve this right away)